sdk/eventhub/Azure.Messaging.EventHubs/stress/src/Configurations/TestConfiguration.cs

```diff
@@ -45,4 +46,25 @@ internal class TestConfiguration
     /// </summary>
     ///
     public int DurationInHours = 150;
+
+    public SHA256 Sha256Hash = default;
```

Comment:
nit:  Doc comments.   :smile:


We should consider making this class disposable and having it own responsibility for cleaning up the hasher.